Task008/main.py

```diff
@@ -0,0 +1,14 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек,
+# если разрешается сделать один разлом по прямой между дольками
+# (то есть разломить шоколадку на два прямоугольника).
+
+# *Пример:*
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n = int(input("Enter n: "))
+m = int(input("Enter m: "))
+k = int(input("Enter k: "))
+
+print("yes" if (k % m == 0) or (k % n == 0) else "no")
```

Comment:
Условия обязательные, но недостаточные. Также необходимо проверить переменную `k` по допустимым значениям. Да, и скобки не нужны.
```suggestion
print("yes" if k % m == 0 or k % n == 0 else "no")
```